tests/data/simple_with_null_values.csv

```diff
@@ -1,10 +1,10 @@
-"id","name","glasses","date","datetime"
-"0","Miro cillik","yes","2020-05-28","2020-05-28 16:00:00"
-"1","Ondrej Hlavacek","","","2020-05-28 16:00:00"
-"2","Martin Halamicek","yes","2020-05-28",""
-"3","Tomas Kacur","yes","",""
-"4","Erik Zigo","","",""
-"5","Marc Raiser","sometimes","",""
-"6","Petr Simecek","yes","",""
-"7","Jakub Matejka","yes","",""
-"8","Ondrej Popelka","","",""
+"id","name","glasses","date","datetime","nullstring"
```

Comment:
Lepsie by to bolo asi nazvat `nullstring` -> `nullablestring`, aby bolo jasne co sa tam testuje.

Reply:
jj přejmenuju to :)